examples/applications/object_detection/retina_net/basic/pascal_voc/train.py

```diff
@@ -0,0 +1,341 @@
+"""
```

Comment:
Should this live in examples/training/object_detection/... instead of examples/applications/object_detection... for consistency with other training scripts?